### DISCUSSION TITLE
erlaubte Sonderzeichen für Passwörter

### DISCUSSION THREAD

**Discussion by ChristophCaina** [OpenCCU/OpenCCU#2003] [🇩🇪 Fragen&Antworten]
Hallo Zusammen,

nur eine kurze Frage:
Gibt es einen bestimmten Grund, warum lediglich die folgenden Sonderzeichen für Passwörter (admin account) erlaubt sind?
```
, !, $, (, ), :, ;, #, ß, Ä, ä, Ö, ö, Ü, ü, -
```
ich würde gerne die volle Palette nutzen - also auch `?, _ ` sofern es hier keine Technische Einschränkung dafür gibt ;)
Vielleicht kann dies ja in irgend einer Art und weise noch implementiert werden - oder stammt diese Einschränkung vom Ursprünglichen Code der CCU ?

**Comment by Baxxy13** [comment]
Die Hintergründe für die Limitierungen kenne ich nicht.
Das Passwort für den jeweiligen User ist aber "Schall und Rauch" für die Wissenden die Zugriff in/zu deinem Netzwerk, und damit zur WebUI der Zentrale, haben.

Das Passwort taugt maximal um den "nicht-Technik-affinen Haus-Teenager" abzuhalten. ;-)

@jmaus kann das bestimmt sachlicher beantworten.

**Reply by jp112sdl** [reply]
`_` sollte meiner Meinung nach gehen.
Alles andere macht die Verarbeitung der Zeichenkette kaputt.
Könnte man sicher verbessern (Vorbild wäre die Behandlung wie beim Setzen einer Text-SV).

Aber Aufwand und Nutzen... @Baxxy13 hat es schon ganz gut beschrieben.
